interpreter/cling/lib/Interpreter/IncrementalParser.h

```diff
@@ -68,9 +71,6 @@ namespace cling {
     // file ID of the memory buffer
     clang::FileID m_VirtualFileID;
 
-    // CI owns it
```

Comment:
I tested root (as I said), well actually I tested root.exe. And you're right, we're intentionally leaking all of that. O well.